nym-vpn-core/README.md

```
@@ -203,10 +204,16 @@ If you omit the `RELEASE` flag or set it to `0`, the binaries will be put into `
 
 For convenience, all build artifacts are also mirrored under `build/` directory in the repo root.
 
-If you want to build for different architecture, pass one of the following parameters to `make`:
+#### Compiling for ARM64 on x64
 
-- `CPU_ARCH=AMD64` to build for x64
-- `CPU_ARCH=ARM64` to build for ARM64
+You can cross-compile from x64 to ARM64, again from a **Visual Studio Developer PowerShell**:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)